### PR TITLE
Cantools recipe

### PR DESCRIPTION
**Pull request by danielhrisca**
re-added cantools recipe

### CODE REVIEW

File: recipes/cantools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - CAN message encoding and decoding.
+    - Simple and extended signal multiplexing.
+    - ``candump`` output decoder.
+    - Node `tester`_.
+  doc_url: http://cantools.readthedocs.io/en/latest/
+  dev_url: https://github.com/eerimoq/cantools
+
+extra:
+  recipe-maintainers:
+    - eerimoq
```
Comment (marcelotrevisani) [inline recipes/cantools/meta.yaml]: @eerimoq are you ok in being a maintainer of this recipe?

Comment (eerimoq) [reply recipes/cantools/meta.yaml]: I would prefer not to as I don't have time, nor intrest, in being so.

Comment (marcelotrevisani) [reply recipes/cantools/meta.yaml]: @danielballan , please remove @eerimoq from this recipe

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks to put this in ``conda-forge``!  :)

**Comment by danielhrisca** [comment]
Thank you for the support Marcelo, and thank you Erik for your fine work.

**Pull request merged**